seacatauth/authn/service.py

```diff
@@ -81,8 +81,11 @@ def __init__(self, app, service_name='seacatauth.AuthenticationService'):
 		self.LoginSessions = {}
 
 		# Metrics - login counters
-		self.LoginFailedCounter = self.MetricsService.create_counter("login_failed", tags={"help": "Counts failed logins."}, init_values={"logins": 0})
-		self.LoginSuccessCounter = self.MetricsService.create_counter("login_success", tags={"help": "Counts successful logins."}, init_values={"logins": 0})
+		self.LoginCounter = self.MetricsService.create_counter(
+			"sca_login",
```

Comment:
`login` is ok, "sca_" prefix is not, we don't use this anywhere officially ... it is either `seacat_auth_login` or just `login`.